### PR TITLE
mail/sms - Ib 7

### LINKED ISSUE #7: mail/sms
send email and sms (if phone number is provided) after registration

send email and sms (if phone number is provided) after acceptation by super admin(related only to hotel owner)

send email and sms (if phone number is provided) after reservation

send email and sms (if phone number is provided) after removal by super admin

send email and sms (if phone number is provided) when users reservation has been removed

send email and sms (if phone number is provided) when there are 5 days left

### PR DESCRIPTION
**Pull request by ibednorz**

### CODE REVIEW

File: services/email/views/owner.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1>Hello {{username}}!</h1>
+<p>You have been pertified as a hotel owner, your hotel is now available to other user.</p>
```
Comment (pdybowski) [inline services/email/views/owner.html]: maybe petrified xdd but i think we want to inform hotel owner that they have been VERIFIED not petrified XD

File: services/email/views/reservation.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<h1>Hello {{username}}!</h1>
+<p>
+    Congratulations, you have just booked your stay at the hotel {{hotel}} .</p>
```
Comment (pdybowski) [inline services/email/views/reservation.html]: remove space between hotel}} and .

File: services/email/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    username: username,
+    hotel: hotel
+  }
+  const htmlToSend = template(replacements)
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: 'bookingcloneapi@gmail.com',
+      pass: 'BOOKINGcloneAPI',
+    },
```
Comment (bartoszbialecki) [inline services/email/index.js:24]: this cannot be in source code

File: services/sms/index.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const Nexmo = require('nexmo')
+
+const nexmo = new Nexmo({
+    apiKey: 'e5c1a352',
+    apiSecret: 'nD9CM2Hf3qjvzVZE',
+  });
```
Comment (bartoszbialecki) [inline services/sms/index.js]: the same here, it cannot be in source code

File: services/email/index.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+const nodemailer = require('nodemailer')
+const handlebars = require('handlebars')
+const fs = require('fs')
+const path = require('path')
+const { emailUser, emailPass } = require('../../config/local.json')
```
Comment (bartoszbialecki) [inline services/email/index.js]: Check how is it used in other places, because I think it should not be used like this, look for config function

**Review by pdybowski — COMMENTED** [review]
add notifyUser function

**Pull request merged**